compatibility-verifier/compCheck.sh

```diff
@@ -157,20 +181,27 @@ fi
 
 # Setup initial cluster with olderCommit and do rolling upgrade
 startServices "$oldTargetDir"
-sleep 20
+#$COMPAT_TESTER pre-controller-upgrade.yaml
 stopService controller "$oldTargetDir"
 startService controller "$newTargetDir"
+#$COMPAT_TESTER pre-broker-upgrade.yaml
 stopService broker "$oldTargetDir"
 startService broker "$newTargetDir"
+#$COMPAT_TESTER pre-server-upgrade.yaml
 stopService server "$oldTargetDir"
 startService server "$newTargetDir"
-sleep 20
+#$COMPAT_TESTER post-server-upgrade.yaml
+
+# Upgrade complated, now do a rollback
 stopService controller "$newTargetDir"
 startService controller "$oldTargetDir"
+#$COMPAT_TESTER post-server-rollback.yaml
 stopService broker "$newTargetDir"
 startService broker "$oldTargetDir"
+#$COMPAT_TESTER post-broker-rollback.yaml
 stopService server "$newTargetDir"
 startService server "$oldTargetDir"
+#$COMPAT_TESTER post-controller-rollback.yaml
 sleep 20
```

Comment:
If you decide to have(or remove) previous `sleep`s, please do the same for this one as well?